site/en/tutorials/generative/dcgan.ipynb

```diff
@@ -332,7 +332,7 @@
         "    model.add(layers.Dropout(0.3))\n",
         "\n",
         "    model.add(layers.Flatten())\n",
-        "    model.add(layers.Dense(1))\n",
+        "    model.add(layers.Dense(1, activation='sigmoid'))\n",
```

Comment:
This is intentional because while this can make the model output more directly interpretable, this approach is discouraged as it's impossible to provide an exact and numerically stable loss calculation for all models when using a softmax output.

See https://www.tensorflow.org/tutorials/quickstart/beginner for the full explanation

Reply:
And you can't change the output activation here without also changing the loss handling later on.

Reply:
> The BCE has from_logits=True.

Oh, I see, that makes sense. sorry for the time waste

Reply:
No problem. Thanks for trying to help out. 👍